panels/models/attraction.py

```diff
@@ -33,6 +34,30 @@ def sluggify(s):
     return RE_SLUG.sub('-', s).lower().strip('-')
 
 
+def treeify(items, keys, value_key=None):
+    if not keys:
+        return items
+    keys = listify(keys)
+    last_key = keys[-1]
+    treeified = OrderedDict()
+    for item in items:
+        current = treeified
+        for key in keys:
+            attr = key(item) if callable(key) else getattr(item, key)
```

Comment:
Do you think it's a good idea to add a default to the `getattr` here, like `None`?

Reply:
I think we want this to raise an `AttributeError` if someone provides the function with an invalid key.